app/models/event.js

```diff
@@ -0,0 +1,34 @@
+var Sequelize = require('sequelize');
+
+var schema = {
+	payload: {
+		type: Sequelize.JSONB,
+		allowNull: false
+	},
+	type: {
+		type: Sequelize.STRING,
+		allowNull: false
+	},
+	dispatchedAt: {
+		type: Sequelize.DATE,
+		allowNull: false,
+		field: 'dispatched_at'
+	},
+};
+
+var options = {
+	underscored: true,
+	tableName: 'event',
+	indexes: [{
+    	name: 'dispatched_at_index',
+    	method: 'BTREE',
+    	fields: ['dispatched_at']
+    }]
+};
+
+module.exports = function(sequelize, DataTypes) {
+	var Event = sequelize.define('Event', schema, options);
+	var Game = sequelize.import(__dirname + '/game');
+	Event.belongsTo(Game);
+	return Event;
+}
```

Comment:
A conversation is a lobby conversation. It can be between two or more people.

A message is a text sent in a lobby conversation. 

(I will document this more later, with the full API.)